etc/shell.nix

```diff
@@ -3,10 +3,15 @@
 
 with import <nixpkgs> {};
 let
-    pinnedSha = "6adf48f53d819a7b6e15672817fa1e78e5f4e84f";
     pinnedNixpkgs = import (builtins.fetchTarball {
-        url = "https://github.com/NixOS/nixpkgs/archive/${pinnedSha}.tar.gz";
+      url = "https://github.com/NixOS/nixpkgs/archive/6adf48f53d819a7b6e15672817fa1e78e5f4e84f.tar.gz";
     }) {};
+
+    # Keep this in sync with rust-toolchain.toml!
```

Comment:
Since rust-toolchain.toml supports comments, maybe it will be helpful to leave a comment about updating shell.nix there as well.

Reply:
done in afd1254c3cf56cd0534e1aebebc6ee79c4499d6d